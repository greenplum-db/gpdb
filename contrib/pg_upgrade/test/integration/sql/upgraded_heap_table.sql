SHOW server_version;

SELECT * FROM users;
