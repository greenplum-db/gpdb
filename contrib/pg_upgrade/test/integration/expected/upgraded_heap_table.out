SHOW server_version;
 server_version 
----------------
 9.4.24         
(1 row)

SELECT * FROM users;
 id | name 
----+------
 1  | Jane 
 2  | John 
 3  | Joe  
(3 rows)
